utils/nctl/sh/scenarios-upgrades/upgrade_scenario_09.sh

```diff
@@ -163,46 +163,51 @@ function _step_05()
     done
 }
 
-# Step 06: Await 1 era.
+# Step 06: Await 1 era and then start node 6 with trusted hash within the allowed range
```

Comment:
I'm wondering if we should also add a node here that starts up with the pre-upgrade hash? Since it would be allowed in the range, we should probably verify that as well.

Reply:
It shouldn't matter if the block is pre or post upgrade if the upgrade was successful. The condition for it to work is to provide a trusted block hash within the allowed range. Actually this is the reason why the test was failing in the past; node 7 was using a hash that was within the allowed range and didn't fail as expected.
So we can start 2 more nodes that have their hash within the allowed range, one pre and one post upgrade to see if it works but not sure it is necessary.